controller/src/db/sqlclient.py

```diff
@@ -58,7 +57,7 @@ def get_local_session(self):
         """
         return self._local_maker()
 
-    def create_tables(self, drop_old: bool = False, names: list = None) -> ApiResponse:
+    def create_tables(self, drop_old: bool = False, names: list = None) -> None:
```

Comment:
no need to hint for returning `None`
```suggestion
    def create_tables(self, drop_old: bool = False, names: list = None):
```